e2e/utils.go

```diff
@@ -127,6 +131,32 @@ func updateSecretForEncryption(c kubernetes.Interface) error {
 	return nil
 }
 
+// updateVaultTokenForEncryption is an hack to update the secrets created by rook to
```

Comment:
*... is a hack ...* indicates that this needs to be changed. Add a `FIXME` or `TODO` describing what is needed.